src/components/PopupWithForm.js

```diff
@@ -28,4 +30,23 @@ export default class PopupWithForm extends Popup {
     this._popupForm.reset();
     super.close();
   }
+
+  setLoading(isLoading) {
```

Comment:
You've implemented the logic for rendering loading via the class methods. Excellent!

Reply:
Thank you!